src/components/home/PageHome.vue

```diff
@@ -1,21 +1,54 @@
 <template>
   <div class="home-page">
-
-    <!--{{> catalog/home/home-banners}}-->
+    <div class="home-banner-option-one">
+      <div class="container-fluid">
+        <div class="row">
+          <BannerTopLeft/>
+          <BannerTopRight/>
+        </div>
+      </div>
+    </div>
+    <div class="home-banner-option-two">
+      <div class="container-fluid">
+        <div class="row">
+          <BannerMiddle/>
+        </div>
+      </div>
+    </div>
+    <div class="home-banner-option-three">
+      <div class="container-fluid">
+        <div class="row">
+          <BannerBottomLeft/>
+          <BannerBottomRight/>
+        </div>
+      </div>
+    </div>
 
     <div class="container">
```

Comment:
I would suggest to hide this `div.container` and anything inside, until we have more sections below.
 Without this div the page looks nicer :)